docs/connector-development/tutorials/building-a-java-destination.md

```diff
@@ -2,24 +2,26 @@
 
 ## Summary
 
-This article provides a checklist for how to create a Java destination. Each step in the checklist has a link to a more detailed explanation below.
+This article provides a checklist for how to create a Java destination. Each step in the checklist
+has a link to a more detailed explanation below.
```

Comment:
why the line break?

Reply:
Most likely a `prettier -w` pass with 100 desired line width. I'm not strongly attached to this approach — I personally like linted markdown to comply to certain width, especially when there are a lot of contibutors, but functionally, same thing.